RenderStateNotation/include/RenderStateNotationParserImpl.hpp

```diff
@@ -73,22 +73,34 @@ class RenderStateNotationParserImpl final : public ObjectBase<IRenderStateNotati
     virtual const RenderStateNotationParserInfo& DILIGENT_CALL_TYPE GetInfo() const override final;
 
 private:
+    struct PipelineHasher
+    {
+        size_t operator()(const std::pair<HashMapStringKey, PIPELINE_TYPE>& Key) const
+        {
+            std::size_t Seed = Key.first.GetHash();
+            HashCombine(Seed, static_cast<size_t>(Key.second));
```

Comment:
You simply can use
`ComputeHash(Key.first.GetHash(), static_cast<size_t>(Key.second))`